### ISSUE TITLE
does router Algorithm has reference paper?

### ISSUE THREAD

**Issue by HBProlove** [vygr/C-PCB#7]
hi vygr,  does router Algorithm has reference paper?

**Comment by vygr** [comment]
Sorry no. I didn’t use any existing paper. Just did something that felt like it would work. :)

I think you could categorise it as a rip up and reroute strategy with a heuristic based on sorting track order by areas. 

Regards

Chris

—————-
Chris Hinsley

**Comment by Summving** [comment]
Yes,  if your project has papers to describe your algorithms will be helpful. as your reply, it seems same as freerouting algorithms  strategy (maybe I am wrong)
